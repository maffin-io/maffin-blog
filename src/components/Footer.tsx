import React from 'react';
import Link from 'next/link';

export default function Footer() {
  return (
    <footer>
      <div className="mt-16 flex flex-col items-center">
        <div className="mb-2 flex space-x-2 text-sm text-gray-500 dark:text-gray-400">
          <Link href="/">Maffin Blog</Link>
          <div>{'\u2022'}</div>
          <div>{`\u00a9${new Date().getFullYear()}`}</div>
        </div>
      </div>
    </footer>
  );
}
