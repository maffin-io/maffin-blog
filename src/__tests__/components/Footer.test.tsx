import { render } from '@testing-library/react';

import Footer from '@/components/Footer';

describe('Footer', () => {
  it('renders as expected', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
