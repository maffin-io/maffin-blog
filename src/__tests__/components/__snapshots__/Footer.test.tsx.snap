// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders as expected 1`] = `
<div>
  <footer>
    <div
      class="mt-16 flex flex-col items-center"
    >
      <div
        class="mb-2 flex space-x-2 text-sm text-gray-500 dark:text-gray-400"
      >
        <a
          href="/"
        >
          Maffin Blog
        </a>
        <div>
          •
        </div>
        <div>
          ©2024
        </div>
      </div>
    </div>
  </footer>
</div>
`;
